cfgov/jinja2/v1/careers/_single.html

```diff
@@ -56,6 +56,7 @@
                 'heading': 'Share this job',
                 'hide_heading': false,
                 'show_linkedin': true,
+                'show_twitter': false,
```

Comment:
Mm, this might be a content question, but I'd guess the job title in `options.text`, `@cfpb` in `options.related`, and `#usajobs` in `options.hashtags`, for the options listed in https://github.com/cfpb/cfgov-refresh/blob/flapjack/cfgov/jinja2/v1/_includes/macros/share.html#L13-L29